content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/serial_numbers.rst

```diff
@@ -1,147 +1,230 @@
+
+.. image:: serial_numbers/serial-numbers-detailed-operations-popup.png
+   :align: center
+   :alt: Serial numbers listed in detailed operations popup.
+
+A :guilabel:`Traceability` smart button will appear upon validating the delivery order. Click the
+:guilabel:`Traceability` smart button to see the updated :guilabel:`Traceability Report`, which
+includes: a :guilabel:`Reference` document; the :guilabel:`Product` being traced; the
+:guilabel:`Lot/Serial #` assigned; and the :guilabel:`Reference` receipt from the previous purchase
+order (if any of the product quantities shared a serial number assigned during receipt of that
+:abbr:`PO (purchase order)`).
+
+Manage serial numbers for different operations types
+====================================================
+
+By default in Odoo, the creation of new serial numbers is only allowed upon *receiving* products
+from a purchase order. Also by default, using *existing* serial numbers isn't allowed upon receipt.
+For sales orders, it is the opposite: new serial numbers cannot be created on the delivery order,
+but existing serial numbers can be used.
+
+To change the ability to use new or existing serial numbers on any operation type, go to
+:menuselection:`Inventory --> Configuration --> Operations Types`, and select the desired
+:guilabel:`Operation Type`. For :guilabel:`Receipts`, the
+:guilabel:`Use Existing Lots/Serial Numbers` option can be enabled by clicking :guilabel:`Edit` and
+then clicking the :guilabel:`checkbox`. For :guilabel:`Delivery Orders`, the
+:guilabel:`Create New Lots/Serial Numbers` option can be enabled by clicking :guilabel:`Edit` and
+clicking the :guilabel:`checkbox`. For any changes made, be sure to :guilabel:`Save`.
+
+.. image:: serial_numbers/serial-numbers-operations-types.png
+   :align: center
+   :alt: Enabled traceability setting in operations type form.
+
+Serial number traceability
 ==========================
 
-Using serial numbers allows you to keep track of where the products were
-received, put in stock, to whom they were sold, and where they were
-shipped to.
-
-Tracking an item is easy: open the *Inventory* app, and go to :menuselection:`Master Data -->
-Lots/Serial Numbers` and click on the serial number corresponding to your search. Then, open the
-*Traceability* information. There, you will see in which documents the serial number has been used.
-
-.. image:: serial_numbers/serial_numbers_12.png
-    :align: center
-
-.. image:: serial_numbers/serial_numbers_13.png
-    :align: center
-
-And, if you want to locate a serial number, you can do so by clicking on
-the *Location* button available on the serial number form.
-
-.. image:: serial_numbers/serial_numbers_14.png
-    :align: center
-
-.. image:: serial_numbers/serial_numbers_15.png
-    :align: center
+The benefit of using serial numbers (or lots) to track products is that it offers end-to-end
```

Comment:
Instead of describing the benefits of serial numbers overall, this can be one sentence explaining the traceability report. "The traceability report shows x, y, and z".